### DISCUSSION TITLE
"defaultCourseFramework" property of the content metadata

### DISCUSSION THREAD

**Discussion by vinukumar-vs** [Sunbird-Knowlg/Community#126] [Feature Requests]
Today the metadata property "defaultCourseFramework" is part of channel read API? 

I hope this is the key property for courses workflows of front-end  application & backend services. Is the naming convention of "course" for the property is suitable for other than learning domain?

`curl --location '{{host}}/api/channel/v1/read/0137541424673095687' \
--header 'Accept: application/json' \
--header 'Authorization: Bearer {{kong-api-key}}'`

**Response:**
{
  ...
  createdOn: "2023-03-16T13:01:40.505+0000"
  defaultCourseFramework: "sunbird_TPD"
  defaultFramework: "NCF"
  ...
}

@Krishnaj20 @pallakartheekreddy @vrayulu @maheshkumargangula

**Comment by pallakartheekreddy** [comment]
@vinukumar-vs After the new collection editor implementation, In workspace while creating any course/Collection editors are not depending on `defaultCourseFramework` or `defaultFramework` properties in channel API.

**Reply by pallakartheekreddy** [reply]
@vinukumar-vs As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
